doc/kernel/usermode/arm_userspace.rst

```diff
@@ -0,0 +1,26 @@
+.. _arm_userspace:
+
+ARM Userspace
```

Comment:
Still need to change this to not be ARM-specific. ARC port is using this too, for example.